specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ManagedEnvironments.json

```diff
@@ -1228,6 +1231,14 @@
             "daprConfiguration": {
               "$ref": "#/definitions/DaprConfiguration",
               "description": "The configuration of Dapr component."
+            },
+            "infrastructureResourceGroup": {
+              "description": "Name of the platform-managed resource group created for the Managed Environment to host infrastructure resources. If a subnet ID is provided, this resource group will be created in the same subscription as the subnet.",
+              "type": "string",
```

Comment:
Seems it is not required, so could we add what will be the default value?

Reply:
If not required, the default is generated based on the environment name and the region, so there is no static default value to list.